sklearn/datasets/_samples_generator.py

```diff
@@ -944,9 +941,6 @@ def make_blobs(
 ):
     """Generate isotropic Gaussian blobs for clustering.
 
-    For an example of usage, see
-    :ref:`sphx_glr_auto_examples_datasets_plot_random_dataset.py`.
-
     Read more in the :ref:`User Guide <sample_generators>`.
```

Comment:
I removed the reference since it is included in the user guide.

Reply:
Yes this is good.